### DISCUSSION TITLE
Capablity to connect multiple Nebula networks using multiple Windows service installation

### DISCUSSION THREAD

**Discussion by zodiaq** [slackhq/nebula#1117] [Ideas]
Now, we can install Nebula in Windows platform as a Windows service.
However, because it always creates a service with the same name "Nebula", I need to run nebula.exe as a process instead when I need to connect another Nebula network.
Could we run multiple Nebula services to connect multiple Nebula networks rather than running individual user processes?
For example, how is it if we could specify the service name when installing the service?
Unless Nebula has a problem when running as multiple services in the same PC, I think it's a useful feature.

**Comment by johnmaguire** [comment]
Hey @zodiaq - I don't know of a simple way to do this currently. I went ahead and created a feature request for this: https://github.com/slackhq/nebula/issues/1118.

I'm closing this thread out so we can try to keep future discussion on the issue. :)
